packages/suite-build/utils/env.ts

```diff
@@ -0,0 +1,13 @@
+const { NODE_ENV, ANALYZE, LAUNCH_ELECTRON, ASSET_PREFIX } = process.env;
+
+const ALLOWED = ['development', 'production'] as const;
+const DEFAULT = ALLOWED[0]; // development
+
+const env = NODE_ENV as typeof ALLOWED[number];
+const environment = NODE_ENV !== undefined && ALLOWED.includes(env) ? env : DEFAULT;
```

Comment:
Could be simplified to
```suggestion
const environment = NODE_ENV === 'production' ? 'production' : 'development';
```

Preserving both the semantics and types.

Reply:
cf1975a